src/platforms/javascript/guides/svelte/config.yml

```diff
@@ -0,0 +1,7 @@
+title: Svelte
```

Comment:
(Don't know where else to put this.) Do we already have plans to upload a logo for svelte in the docs?

Reply:
Haven't done it yet but we either do it when we first publish the docs or when we go into beta. Still need to take a look at how we did it with Remix.